planner/core/src/main/java/eu/seaclouds/platform/planner/core/DamGenerator.java

```diff
@@ -75,10 +74,22 @@
     public static final String SEACLOUDS_APPLICATION_INFORMATION_POLICY_TYPE = "seaclouds.policies.app.information";
     public static final String SEACLOUDS_APPLICATION_POLICY_NAME = "seaclouds.app.information";
     public static final String SEACLOUDS_NODE_PREFIX = "seaclouds.nodes";
+    public static final String SEACLOUDS_NODE_TYPES = "seaclouds-types";
+    public static final String SEACLOUDS_NODE_TYPES_VERSION = "0.8.0-SNAPSHOT";
 
     private static DeployerTypesResolver deployerTypesResolver;
     static Map<String, MonitoringInfo> monitoringInfoByApplication=new HashMap<>();
 
+    /**
+     *
+     * @param adp YAML string containing the ADP
```

Comment:
I am not sure if current ADP can be managed as a [ArchiveRoot](https://github.com/alien4cloud/alien4cloud/blob/d802eef4cc3cd7416155d4716ecb79e1de861dd8/alien4cloud-core/src/main/java/alien4cloud/tosca/model/ArchiveRoot.java) but I think we have to try